cmd/kind/delete/cluster/deletecluster.go

```diff
@@ -42,11 +45,24 @@ func NewCommand() *cobra.Command {
 			return runE(flags, cmd, args)
 		},
 	}
-	cmd.Flags().StringVar(&flags.Name, "name", "1", "the cluster name")
+	cmd.Flags().StringVar(&flags.Name, "name", cluster.DefaultName, "the cluster name")
+	cmd.Flags().BoolVar(&flags.Force, "force", false, "discard the check if a cluster exists")
 	return cmd
 }
 
 func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
+	// Check if the cluster name exists
+	if !flags.Force {
```

Comment:
if `create` was called but `ctrl+c`-ed immediately it will create a context with that name, but `delete` will not work on it.
so the clusters is in a weird state where both `create` and `delete` don't work on the name.

adding this `force` makes it possible to skip the check and force the deletion.
opened to ideas, but might remove this check.

adding TODO for `[y/n]` prompt` might be good too.

Reply:
> but delete will not work on it.

hmm? how so? delete will delete any tagged containers, and we don't leave anything behind without containers ...?